internal/kafka/internal/workers/kafka_mgrs/preparing_kafkas_mgr.go

```diff
@@ -25,13 +25,14 @@ type PreparingKafkaManager struct {
 }
 
 // NewPreparingKafkaManager creates a new kafka manager
-func NewPreparingKafkaManager(kafkaService services.KafkaService, bus signalbus.SignalBus) *PreparingKafkaManager {
+func NewPreparingKafkaManager(kafkaService services.KafkaService, bus signalbus.SignalBus, ReconcilerConfig *workers.ReconcilerConfig) *PreparingKafkaManager {
```

Comment:
```suggestion
func NewPreparingKafkaManager(kafkaService services.KafkaService, bus signalbus.SignalBus, reconcilerConfig *workers.ReconcilerConfig) *PreparingKafkaManager {
```

let's make the argument start with a small case letter throughout where the `ReconcilerConfig *workers.ReconcilerConfig` parameter is added .

Reply:
Brilliant :) I've pushed the changes, thanks @machi1990